core/server/middleware/serve-favicon.js

```diff
@@ -60,7 +60,7 @@ function serveFavicon() {
                     res.end(content.body);
                 });
             } else {
-                filePath = path.join(config.get('paths:corePath'), 'shared', 'favicon.ico');
+                filePath = path.join(config.get('paths:corePath'), 'server/public', 'favicon.ico');
```

Comment:
Would it make sense actually, to have a new config path property for the public assets here? e. g. `assetPath`?